microceph/cmd/microcephd/main.go

```diff
@@ -122,7 +122,7 @@ func main() {
 
 	app.PersistentFlags().BoolVarP(&daemonCmd.global.flagHelp, "help", "h", false, "Print help")
 	app.PersistentFlags().BoolVar(&daemonCmd.global.flagVersion, "version", false, "Print version number")
-	app.PersistentFlags().BoolVarP(&daemonCmd.global.flagLogDebug, "debug", "d", false, "Show all debug messages")
+	app.PersistentFlags().BoolVarP(&daemonCmd.global.flagLogDebug, "debug", "d", true, "Show all debug messages")
```

Comment:
Don't debug log in prod code

Reply:
yeah my original changes were not rebased on your logger changes so i had to do this dance for debugging.

Reply:
let me disable it again.